tests/views_integration/Tribe/Events/Views/V2/Partials/List_View/Month_SeparatorTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Tribe\Events\Views\V2\Partials\List_View\Event;
+
+use Tribe\Test\PHPUnit\Traits\With_Post_Remapping;
+use Tribe\Test\Products\WPBrowser\Views\V2\HtmlPartialTestCase;
+
+class Month_SeparatorTest extends HtmlPartialTestCase
+{
+	use With_Post_Remapping;
+
+	protected $partial_path = 'list/month-separator';
+
+	/**
+	 * Test should render month separator
+	 */
+	public function test_should_render_month_separator() {
+		$event_1 = $this->mock_event( 'events/single/1.json' )->get();
```

Comment:
Ok, I think you got the idea: please review your usages of the method and, where you're not modifying the template in any way, use `get_mock_event`; there's a small performance gain that is worth a search and replace.